Panic on wrong arguments

#### Affected Version
yay v10.0.0 - libalpm v12.0.1

#### Issue


Catch panic while trying to search. 
#### Steps to reproduce
<!-- Use exact commands where applicable -->
1. Type `yay svne coo` for example
2. Catch the error `panic: runtime error: index out of range [1] with length 1`

#### Output
```
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
main.narrowSearch(0xc0000a1400, 0x2, 0x2, 0xc0000a1401, 0x2, 0x2, 0xc0000aa5c8, 0x40afe5, 0xc0000c5c98)
        github.com/Jguer/yay/v10/query.go:177 +0x770
main.displayNumberMenu(0xc0000a1400, 0x2, 0x2, 0x2, 0x0)
        github.com/Jguer/yay/v10/cmd.go:307 +0xac
main.handleYogurt(...)
        github.com/Jguer/yay/v10/cmd.go:245
main.handleYay(0x7ea281, 0x1)
        github.com/Jguer/yay/v10/cmd.go:234 +0x179
main.handleCmd(0x0, 0x0)
        github.com/Jguer/yay/v10/cmd.go:172 +0x1c0
main.main()
        github.com/Jguer/yay/v10/main.go:243 +0xf0
```

--- a/query.go
+++ b/query.go
@@ -169,12 +169,12 @@ func narrowSearch(pkgS []string, sortS bool) (aurQuery, error) {
 
 	for i := range r {
 		match := true
-		for i, pkgN := range pkgS {
-			if usedIndex == i {
+		for j, pkgN := range pkgS {
+			if usedIndex == j {
 				continue
 			}
 
-			if !(strings.Contains(r[i].Name, pkgN) || strings.Contains(strings.ToLower(r[i].Description), pkgN)) {
+			if !(strings.Contains(r[j].Name, pkgN) || strings.Contains(strings.ToLower(r[j].Description), pkgN)) {
 				match = false
 				break
 			}

